app/lib/common/pages/fatal_fail.dart

```diff
@@ -124,4 +108,15 @@ class _FatalFailPageState extends ConsumerState<FatalFailPage> {
       ),
     );
   }
+
+  void onCopy() {
+    Clipboard.setData(
+      ClipboardData(text: '${widget.error}\n$stack'),
+    );
+    EasyLoading.showToast('Error & Stacktrace copied to clipboard');
```

Comment:
Copying error and stacktrace isn't helpful for debugging and reporting evidence rather, I would just simply show it on the UI with ```EasyLoading.showError()```.

Reply:
@gtalha07 disagree. this is the features as intended.